doc/conf.py

```diff
@@ -1,13 +1,14 @@
-"""Sphinx configuration file for an LSST stack package.
-
-This configuration only affects single-package Sphinx documentation builds.
-"""
+"""Sphinx configuration file for TSSW package"""
 
 from documenteer.sphinxconfig.stackconf import build_package_configs
 import lsst.ts.dsm
 
 
 _g = globals()
-_g.update(build_package_configs(
-    project_name='ts_dsm',
-    version=lsst.ts.dsm.version.__version__))
+_g.update(
+    build_package_configs(
+        project_name="ts_dsm", version=lsst.ts.dsm.version.__version__
```

Comment:
I strongly recommend using `lsst.ts.dsm.__version__` so you don't need the `version.py` file.

Reply:
Done.